cli/revoke/revoke.go

```diff
@@ -0,0 +1,64 @@
+// Package revoke implements the revoke command.
+package revoke
+
+import (
+	"database/sql"
+	goerr "errors"
```

Comment:
Removing the alias exposed golint nits about punctuation in error strings - looks like it only looks for `errors.New` and `fmt.Errorf`